library/std/src/sync/mpmc/mod.rs

```diff
@@ -1,8 +1,102 @@
-//! Multi-producer multi-consumer channels.
+//! Multi-producer, multi-consumer FIFO queue communication primitives.
+//!
+//! This module provides message-based communication over channels, concretely
+//! defined among three types:
+//!
+//! * [`Sender`]
+//! * [`Receiver`]
+//!
+//! A [`Sender`] is used to send data to a set of [`Receiver`]s. Both
+//! sender and receiver are clone-able (multi-producer) such that many threads can send
+//! simultaneously to receivers (multi-consumer).
```

Comment:
```suggestion
//! [`Sender`]s are used to send data to a set of [`Receiver`]s. Both
//! sender and receiver are cloneable (multi-producer) such that many threads can send
//! simultaneously to receivers (multi-consumer).
```

Pluralize since there are multiple senders, and "cloneable" is usually one word

Reply:
I find the placement of (multi-producer) after "Both are cloneable" odd, I think having that after "many threads can send" would make more sense
```suggestion
//! A [`Sender`] is used to send data to a set of [`Receiver`]s. Both
//! sender and receiver are clone-able such that many threads can send (multi-producer)
//! simultaneously to receivers (multi-consumer).
```